core/domain/prod_validation_jobs_one_off.py

```diff
@@ -732,7 +732,9 @@ def map(model_instance):
 
     @staticmethod
     def reduce(key, values):
-        if len(values) > 1:
+        # If normalized name is not set, we do not comapare it with normalized
```

Comment:
Explain why it is valid for the normalized name to not be set.

Reply:
Done.